src/scroll_list/ScrollList.js

```diff
@@ -1066,6 +1066,7 @@ define(function(require) {
                     panToOptions,
                     offset,
                     itemLayout.scaleToFit,
+                    (itemLayout.left || 0 + itemLayout.paddingLeft || 0),
```

Comment:
The 0 + itemLayout.paddingLeft looks goofy.  Can you just use itemLayout.paddingLeft?